packages/web-client/src/helpers/share/role.ts

```diff
@@ -110,11 +110,11 @@ export class SpaceShareRole extends ShareRole {
   public description(allowSharing: boolean): string {
     switch (this.name) {
       case spaceRoleViewer.name:
-        return $gettext('Download and preview')
+        return $gettext('Recipients can view and download.')
```

Comment:
With `Recipients` we are not doing ourselves any favor in terms of gender neutral german language `Empfangende Personen`. If we don't find a better solution I'd just list the permissions without the Recipients can...

Reply:
Empfänger:innen :trollface:

Reply:
The official, Duden-compliant gender wording is "Empfänger/-innen", not "empfangende Personen". This is also confirmed from customer side. IMO we should use this form if no real gender-neutral term is available.
FYI: This was my basis for reviewing all german translations for ocis and web repos in TX.